docs/node-mixin/dashboards/use.libsonnet

```diff
@@ -53,8 +53,8 @@ local g = import 'grafana-builder/grafana.libsonnet';
           { yaxes: g.yaxes({ format: 'percentunit', max: 1 }) },
         )
         .addPanel(
-          g.panel('Memory Saturation (Swapped Pages)') +
-          g.queryPanel('instance:node_memory_swap_io_pages:rate1m{%(nodeExporterSelector)s}' % $._config, '{{instance}}', legendLink) +
+          g.panel('Memory Saturation') +
```

Comment:
As this is just an approximation to "real" memory saturation (whatever that may be), perhaps have a note in parentheses pointing to what we are actually graphing here (similar to what we had before), e.g.

`g.panel('Memory Saturation (Major Page Faults)') +`

Reply:
Makes sense, added.